Automattic/wp-calypso
# Plugins: Hide wp-admin Action Links if there's a Calypso UI

Hide the wp-admin action links buttons (that point to wp-admin settings screens) if an extension's `package.json` sets `section.settings_path`.

As requested by @dbtlr in Slack. Sort of a follow-up to #14422; also related to p6q6po-9eH-p2#comment-87711.

Before:

![image](https://user-images.githubusercontent.com/96308/29618918-e2d47502-8819-11e7-9aab-4dad8fa6e6a1.png)

After (hint: the 'Settings' button is gone):

![image](https://user-images.githubusercontent.com/96308/29673009-371e979e-88ee-11e7-95f6-9d90afdec7c4.png)

To test:
* You need a JP site with the WP Super Cache plugin installed and active.
* Navigate to `calypso.localhost:3000/plugins/wp-super-cache/<yourJPsite>`.
* Verify that there's no 'Settings' button.
* You can also check other extensions that set `section.settings_path`, including WP Job Manager and Zoninator.
* Verify that the button is still visible for extensions that _don't_ set `section.settings_path` in their 
`package.json`. Right now, that's only WooCommerce (the button is labelled 'WP Admin' there).
